lib/fpm/package/pacman.rb

```diff
@@ -0,0 +1,394 @@
+# -*- coding: utf-8 -*-
+require "fpm/package"
+require "backports"
+require "fileutils"
+require "find"
+
+class FPM::Package::Pacman < FPM::Package
+
+  opt_depends = []
+  option "--optdepends", "PACKAGE",
```

Comment:
subjective - Is `--pacman-optdepends` a good name for the flag? Thoughts on `--pacman-optional-depends`  or something not abbreviated? It's your call :)